lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -475,12 +422,11 @@ def guest_address(vm)
   end
 
   def wait_for_access(connect_host, connect_port, protocol)
-    if protocol == "winrm"
-      load_winrm_deps
-      if get_config(:winrm_transport) == "ssl" && get_config(:winrm_port) == "5985"
-        config[:winrm_port] = "5986"
+    if winrm?
+      if get_config(:connection_transport) == "ssl" && get_config(:connection_port) == "5985"
```

Comment:
`connection_transport` option is not available it should be `--winrm-ssl`?

Reply:
Changed the option to winrm_ssl.